google/cloud/storage/blob.py

```diff
@@ -2386,7 +2386,7 @@ def _do_upload(
 
         return response.json()
 
-    def upload_from_file(
+    def _prep_and_do_upload(
```

Comment:
Let's just call this _do_upload since there's no other method we're disambiguating with in this case.

Reply:
Correction: looks like we already have one of those so this is fine as it is.